spec/unit/files/servicenow_spec.rb

```diff
@@ -121,8 +124,42 @@
       certname = 'example.puppet.com'
       uri = "https://#{config['instance']}.service-now.com/api/now/table/#{config['table']}?#{config['certname_field']}=#{certname}&sysparm_display_value=true"
 
-      expect(ServiceNowRequest).to receive(:new).with(uri, 'Get', nil, config['user'], config['password'])
+      expect(ServiceNowRequest).to receive(:new).with(uri, 'Get', nil, 'admin', 'password')
```

Comment:
This change was b/c of a stupid Ruby gotcha. Apparently `[]` works with strings, so e..g `"foo"['bar']` works. That means if we mock `config` wrong (so that it becomes a String), this test could give us a false positive (and that's exactly what happened when I was debugging my hiera-eyaml unit test). Figured I may as well make this test more secure.